generator/src/generators/two-fer.cr

```diff
@@ -0,0 +1,42 @@
+require "../exercise_generator"
+require "../exercise_test_case"
+
+class TwoFerGenerator < ExerciseGenerator
+  def exercise_name
+    "two-fer"
+  end
+
+  def test_cases
+    JSON.parse(data)["cases"].as_a.map do |test_case|
+      TwoFerTestCase.new(test_case)
+    end
+  end
+end
+
+class TwoFerTestCase < ExerciseTestCase
+  private getter description : JSON::Any
+  private getter name : JSON::Any
+  private getter expected : JSON::Any
+
+  def initialize(test_case)
+    @description = test_case["description"]
+    @name = test_case["input"]["name"]
+    @expected = test_case["expected"]
+  end
+
+  def getName(input)
+    if input.as_s? == nil
+        ""
+    else
+        "(\"#{name}\")"
+    end
+  end
+
+  def workload
+    "TwoFer.twoFer#{getName(name)}.should eq(\"#{expected}\")"
```

Comment:
It is standard in crystal to use `snake_case` for method names. So `twoFer` should be `two_fer`

In addition, could we just do this instead of delegating the name to a method?
`"TwoFer.twoFer(\"#{@name}\").should eq(\"#{expected}\")"`

Reply:
Ahh I see the issue, I think instead of `getName` maybe we should call the method `test_case_argument` or something which would be a bit more descriptive

Reply:
Ah, I completely missed this on the last PR for the armstrong exercise. @amscotti , if you could edit that spec and example at some point, that'd be awesome. Otherwise I could update it.

Reply:
Sounds good, I'll make the updates for both tonight.